src/ol/source/DataTile.js

```diff
@@ -31,7 +31,7 @@ import {toPromise} from '../functions.js';
  * @property {boolean} [opaque=false] Whether the layer is opaque.
  * @property {import("./State.js").default} [state] The source state.
  * @property {number} [tilePixelRatio] Tile pixel ratio.
- * @property {boolean} [wrapX=true] Render tiles beyond the antimeridian.
+ * @property {boolean} [wrapX=false] Render tiles beyond the antimeridian.
```

Comment:
This is a documentation bug.  There was no default applied by this class, and the parent class defaults to false.